draft-ietf-tls-dtls13.md

```diff
@@ -1128,11 +1128,15 @@ the length field of the original message.  An unfragmented message is
 a degenerate case with fragment_offset=0 and fragment_length=length.
 Each range MUST be delivered in a single UDP datagram.
 
-When a DTLS implementation receives a handshake message fragment, it
-MUST buffer it until it has the entire handshake message.  DTLS
+When a DTLS implementation receives a handshake message fragment corresponding
+to the next expected handshake message sequence number, it
+MUST buffer it until it has the entire handshake message. DTLS
```

Comment:
Are you making this extra clarification to keep the buffer size smaller? 

I think you are saying that when a client receives a fragment of a ServerHello message and a fragment of an EncryptedExtensions message then it needs to buffer the ServerHello message fragment (assuming that's the next message it has not yet received) while it may discard the fragment of the EncryptedExtensions message. Correct?

Reply:
That's correct. This has been the rule for some time, it's just that we managed to provide contradictory guidance.